lib/connection.ex

```diff
@@ -519,6 +520,27 @@ defmodule Janus.Connection do
     handle_successful_payload(transaction, payload, state)
   end
 
+  defp handle_payload(
+         %{
+           "janus" => "event",
+           "transaction" => transaction,
+           "plugindata" => %{
+             "plugin" => _plugin,
+             "data" => data
+           },
+           "jsep" => jsep,
+           "sender" => sender
```

Comment:
As mentioned in person, this fields will only be included in some requests, so we cannot match on them here

Reply:
I've decided to stick to the Janus original payload shape and only filter out transaction. Modifying the returned payload has to be done in a bigger refactor allowing to document such shape. See #17